hudi-client/hudi-client-common/src/main/java/org/apache/hudi/index/bucket/BucketIdentifier.java

```diff
@@ -91,6 +91,10 @@ public static String newBucketFileIdPrefix(int bucketId) {
     return newBucketFileIdPrefix(bucketIdStr(bucketId));
   }
 
+  public static String newBucketFileIdPrefix(String fileId, int bucketId) {
```

Comment:
This method is optimized with the other two methods `newBucketFileIdPrefix(int ​​bucketId)` and `newBucketFileIdPrefix(String bucketId)`?

Reply:
No, newBucketFileIdPrefix(String fileId, int bucketId) function can make BulkInsertBucketInternalWriterHelper use the super function getNextFileId() of BulkInsertDataInternalWriterHelper. if we try to use the newBucketFileIdPrefix(String bucketId), we need to modify the getNextFileId() in BulkInsertDataInternalWriterHelper.